packages/backend/src/api/factory.ts

```diff
@@ -31,6 +32,7 @@ export function createBackendApiClient(options: CreateBackendApiOptions) {
     clients: new ClientAPI(request),
     emailAddresses: new EmailAddressAPI(request),
     invitations: new InvitationAPI(request),
+    __internal_machineTokens: new MachineTokensAPI(request),
```

Comment:
_Disclaimer, I am not familiar with this codebase_

Why is this __internal_machineTokens?

Reply:
It's internal until we release it GA